plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -964,4 +969,179 @@ private static boolean isNumericType(Type type)
                 type.equals(DOUBLE) || type.equals(REAL) ||
                 type instanceof DecimalType;
     }
+
+    static class SparkMetastoreUtil
```

Comment:
Extract this class from `ThriftMetastoreUtil`

Reply:
This would require extracting also `StringParseUtil` to be available for both `ThriftMetastoreUtil` and `SparkMetastoreUtil`.